db/deploy/mutations/parse_gis_data.sql

```diff
@@ -0,0 +1,47 @@
+-- Deploy ccbc:mutations/parse_gis_data to pg
+
+BEGIN;
+
+create or replace function ccbc_public.parse_gis_data(batchId int)
+returns text as 
+$function$
+
+declare
+  application_id int;
+  ccbc_id text;
+  json_row jsonb;
+  json_file jsonb;
+  new_row_id int;
+  result text;
+begin
+    result := null;
+
+    -- load JSON data
+    select json_data into json_file  from ccbc_public.gis_data where id = batchId;
+
+    -- Loop over each object in the JSON array
+    for json_row in select * from jsonb_array_elements(json_file)
+    loop
+        ccbc_id := json_row->>'ccbc_number'; 
+
+        -- Look up the application_id based on the ccbc_number value
+        select id into application_id from ccbc_public.application where ccbc_number = ccbc_id;
+        -- If no row was found, add to list 
+        if not found then
+            result := coalesce(result || ',','') || ccbc_id; 
```

Comment:
Are we planning on saving even if the file has non-valid `ccbc_id`s?

Reply:
Yes, file already saved anyway (in `gis_data` table. List of non-valid ccbc_ids returned from the function, so UI can display if needed. I used Prod JSON in Test environment, and, as expected, some ccbc_id were not found.